src/main/java/uk/gov/hmcts/ccd/domain/service/validate/DefaultValidateCaseFieldsOperation.java

```diff
@@ -61,68 +59,64 @@
 
     private void validateOrganisationPolicy(String caseTypeId, CaseDataContent content) {
         final List<String> errorList = new ArrayList<>();
-        // if there is not a default value. it means that there will not be organisation policy validation.
-        // hence if there is at least one ORGANISATION_POLICY_ROLE all default value logic will be executed.
-        if (!isOrganisationPolicyLogicInTheContent(content)) {
-            return;
-        }
-        caseDefinitionRepository.getCaseType(caseTypeId).getEvents().stream().filter(
-            event -> event.getId().equals(content.getEventId())
-        ).forEach(
-            caseEventDefinition -> caseEventDefinition.getCaseFields().stream().forEach(
-                caseField -> caseField.getCaseEventFieldComplexDefinitions().stream().filter(
-                    caseEventFieldComplexDefinition -> {
-                        if (caseEventFieldComplexDefinition.getReference().equals(ORGANISATION_POLICY_ROLE)) {
-                            //get extract the default value  from the content for the current caseField
-                            final Optional<String> caseFieldDefaultValue = getDefaultValueFromContentByCaseFieldID(content, caseField.getCaseFieldId());
-                            validateOrgPolicyCaseAssignedRole(
-                                caseEventFieldComplexDefinition,
-                                caseFieldDefaultValue,
-                                caseField.getCaseFieldId(),
-                                errorList);
-                            return true;
-                        } else {
-                            return false;
-                        }
-                    }
-                ).collect(Collectors.toList()))
-        );
+        caseDefinitionRepository.getCaseType(caseTypeId).getEvents().stream()
+            .filter(event -> event.getId().equals(content.getEventId()))
+            .forEach(caseEventDefinition -> caseEventDefinition.getCaseFields()
```

Comment:
`CaseTypeDefinition` has a `findCaseEvent()` method returning `Optional<CaseEventDefinition>`.

```suggestion
        caseDefinitionRepository.getCaseType(caseTypeId).findCaseEvent(content.getEventId())
            .ifPresent(caseEventDefinition -> caseEventDefinition.getCaseFields()
```

Reply:
changed to the suggested